src/onnx/parse_pad.cpp

```diff
@@ -177,11 +177,14 @@ struct parse_pad : op_parser<parse_pad>
                 MIGRAPHX_THROW("PARSE_PAD: input `value` must be constant");
             }
             auto val_arg = val_ins->eval();
-            if(val_arg.get_shape().elements() != 1)
+            if(val_arg.get_shape().elements() > 1)
```

Comment:
When there are more than 1 pad values, do they all have same value ?

Reply:
Yes, the `const_value` is just a scalar value. If there are more than 1 pad values they all use the same `const_value`.